src/test/java/ro/unibuc/triplea/domain/games/steam/utils/IdentifierUtilTest.java

```diff
@@ -0,0 +1,58 @@
+package ro.unibuc.triplea.domain.games.steam.utils;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class IdentifierUtilTest {
+
+    @Test
+    public void testIsPositiveNumeric_ValidNumericString() {
+        String numericString = "12345";
+        assertTrue(IdentifierUtil.isPositiveNumeric(numericString));
+    }
+
+    @Test
+    public void testIsPositiveNumeric_InvalidNumericString() {
+        String nonNumericString = "abc123";
+        assertFalse(IdentifierUtil.isPositiveNumeric(nonNumericString));
+    }
+
+    @Test
+    public void testIsPositiveNumeric_NegativeNumber() {
+        String negativeNumber = "-12345";
+        assertFalse(IdentifierUtil.isPositiveNumeric(negativeNumber));
+    }
+
+    @Test
+    public void testIsPositiveNumeric_LeadingSpaces() {
+        String leadingSpaces = "   12345";
+        assertFalse(IdentifierUtil.isPositiveNumeric(leadingSpaces));
+    }
+
+    @Test
+    public void testIsNumeric_TrailingSpaces() {
+        String trailingSpaces = "12345   ";
+        assertFalse(IdentifierUtil.isPositiveNumeric(trailingSpaces));
+    }
+
+    @Test
+    public void testIsPositiveNumeric_LeadingAndTrailingSpaces() {
+        String spacesAroundNumber = "  12345   ";
+        assertFalse(IdentifierUtil.isPositiveNumeric(spacesAroundNumber));
+    }
+
```

Comment:
We don't necessarily need to change this now, but FYI for these kinds of methods there are ways to create a single test for multiple parameters at once, via parameterized tests - [see here](https://www.baeldung.com/parameterized-tests-junit-5). All the test cases for the method could be covered by only one test, in this way. However you could also make the argument that having multiple separate tests allows for cleaner separation.

Reply:
Thanks for letting me know. First i considered to let the trailing spaces to be valid, but i think the actual form is better.